core/web/router.go

```diff
@@ -189,9 +189,8 @@ func metricRoutes(r *gin.RouterGroup, includeHeap bool) {
 	pprofGroup.GET("/allocs", ginHandlerFromHTTP(pprof.Handler("allocs").ServeHTTP))
 	pprofGroup.GET("/block", ginHandlerFromHTTP(pprof.Handler("block").ServeHTTP))
 	pprofGroup.GET("/goroutine", ginHandlerFromHTTP(pprof.Handler("goroutine").ServeHTTP))
-	if includeHeap {
```

Comment:
is it a testing artifact?

Reply:
temporarily change that I made for the soak test image, removing now